src/Components/ContactsList.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import "./ContactsList.css";
+import ContactForm from "./ContactForm.js";
+
+const ContactsList = () => {
+  const [contacts, setContacts] = useState([
+    {
+      firstName: "Ivan",
+      lastName: "Yakushenko",
+      phoneNumber: "+380934869702",
+    },
+    {
+      firstName: "Oleg",
+      lastName: "Bobrov",
+      phoneNumber: "+380973358892",
+    },
+    {
+      firstName: "Irina",
+      lastName: "Rogovskaya",
+      phoneNumber: "+380956699944",
+    },
+  ]);
+
+  const [show, setShow] = useState("true")
+
+  const addContact = (firstName, lastName, phoneNumber) => setContacts([...contacts, { firstName, lastName, phoneNumber }]);
+
+  const toggleShow = () => {
+    return setShow(!show);
+  };
+
+  const removeContact = (index) => {
+    const newContacts = [...contacts];
+    newContacts.splice(index, 1);
+    setContacts(newContacts);
+  };
+
+  return (
+    <div className="todo-list">
+      {contacts.map((contact, index) => (
+        <div key={contact.phoneNumber} className="todo">
```

Comment:
I would suggest you create a separate component for this part of the code

```
function ContactItem(props) {
  const handleRemove = () => {
    // I would suggest you work with id rather than with an index
    props.onRemove(props.id)
  }

  return (
    <div className="todo">
      <span className="todo">
        {props.item.firstName}
      </span>
      <span className="todo">
        {props.item.lastName}
      </span>
      <span className="todo">
        {props.item.phoneNumber}
      </span>
      <button onClick={handleRemove}>
        Delete Contact
      </button>
    </div>
  )
}
```